iaso/models/org_unit.py

```diff
@@ -689,7 +689,17 @@ def __str__(self) -> str:
 
     def save(self, *args, **kwargs):
         self.clean()
+        # Save old values.
+        self.old_parent = self.org_unit.parent
+        self.old_name = self.org_unit.name
+        self.old_org_unit_type = self.org_unit.org_unit_type
+        self.old_location = self.org_unit.location
+        self.old_opening_date = self.org_unit.opening_date
+        self.old_closed_date = self.org_unit.closed_date
```

Comment:
This is taking the values of the org unit after the changes have been saved, so we will not see the difference for approved changes